ckan/logic/action/create.py

```diff
@@ -1302,7 +1302,7 @@ def tag_create(context, data_dict):
     :py:func:`~ckan.logic.action.update.package_update` function.)
 
     :param name: the name for the new tag, a string between 2 and 100
-        characters long containing only alphanumeric characters and ``-``,
+        characters long containing only alphanumeric characters, spaces and the characters ``-``,
```

Comment:
Sorry @Gauravp-NEC, one more thing to make the tests pass:

```suggestion
        characters long containing only alphanumeric characters, 
        spaces and the characters ``-``,
```